### PR TITLE
Gives Sleeper Agents Listening Outpost Access

### PR DESCRIPTION
**Pull request by cheekybrdy**
[BALANCE] [INPUT WANTED] [FEATURE]

## About the PR 
Gives sleepers a implant with listening post access on sleeper roll.

## Why's this needed? 
Adds Listening Post Access to sleepers. This gives them the ability to get access to some traitor items, without adding the antag power of a full uplink to them.

## Changelog 

```changelog
(u)Cheekybrdy
(*)Added listening post access to sleeper agents, this is done through a implant temporarily.
```

### CODE REVIEW

File: code/modules/antagonists/traitor/sleeper_agent.dm [hunk 1]
```diff
@@ -5,7 +5,10 @@
 	antagonist_icon = "traitor"
 
 /datum/antagonist/sleeper_agent/announce()
+	var/mob/living/carbon/human/H = src.owner.current
 	boutput(owner.current, "<h3><span class='alert'>You have awakened as a Syndicate [display_name]!</span></h3>")
+	new /obj/item/implant/access/infinite/sleeper(H)
+	boutput(owner.current, "<h3><span class='alert'>You have been granted access to the local listening post due to a new implant, don't lose it!</span></h3>")
```
Comment (Mister-Moriarty) [inline code/modules/antagonists/traitor/sleeper_agent.dm:11]:
```suggestion
	var/mob/living/carbon/human/H = src.owner.current
	boutput(H, "<h3><span class='alert'>You have awakened as a Syndicate [display_name]!</span></h3>")
	new /obj/item/implant/access/infinite/sleeper(H)
	boutput(H, "<h3><span class='alert'>You have been granted access to the local listening post due to a new implant, don't lose it!</span></h3>")
```
Minor style critique.

**Review by Mister-Moriarty — CHANGES_REQUESTED** [review]

File: code/modules/antagonists/traitor/sleeper_agent.dm (hunk 1 above)
[the remark this answers is not in the document]
Comment (BatElite) [reply code/modules/antagonists/traitor/sleeper_agent.dm:11]: Agreeing with this. My personal suggestion would be "An implant within you pings awake,[...]" so as to not imply that the syndicate is actively monitoring folks' implants, but that's a minor thing.

File: code/modules/antagonists/traitor/sleeper_agent.dm [hunk 2]
```diff
@@ -5,7 +5,10 @@
 	antagonist_icon = "traitor"
 
 /datum/antagonist/sleeper_agent/announce()
-	boutput(owner.current, "<h3><span class='alert'>You have awakened as a Syndicate [display_name]!</span></h3>")
+	var/mob/living/carbon/human/H = src.owner.current
+	boutput(H, "<h3><span class='alert'>You have awakened as a Syndicate [display_name]!</span></h3>")
+	new /obj/item/implant/access/infinite/sleeper(H)
+	boutput(owner.current, "You feel something activate within you, and a vision of a place hidden in the asteroids near the station.")
```
Comment (Mister-Moriarty) [inline code/modules/antagonists/traitor/sleeper_agent.dm:11]:
```suggestion
	boutput(H, "You feel something activate within you, and a vision of a place hidden in the asteroids near the station.")
```
`owner.current` got reverted in what I assume was a merge conflict between your origin and local branches. Ensure to pull changes from your origin prior to making changes on your local.

I also feel that the text is now too ambiguous.

**Comment by cheekybrdy** [comment]
As of this, it adds the fingerpad discussed in the forums into the code but not on any map. It also currently has the implant still in it for as the fingerpads get added to maps, should I take it out entirely?

**Comment by TheColorCyan** [comment]
Its probably better if you put fingerpads on all listening posts and remove implants entirely 
Also this is a buff to traitors and spiefs as well since they dont have to get an agent card

**Comment by cheekybrdy** [comment]
> Its probably better if you put fingerpads on all listening posts and remove implants entirely Also this is a buff to traitors and spiefs as well since they dont have to get an agent card

I agree completely, I'm just not sure if I should have implants in to start and have map PRs for the handpads for merge conflict purposes. Then remove implants once those are in or just remove them from this first PR entirely.

**Comment by TheColorCyan** [comment]
> I agree completely, I'm just not sure if I should have implants in to start and have map PRs for the handpads for merge conflict purposes. Then remove implants once those are in or just remove them from this first PR entirely.

You can just add them all in this PR and fully remove implants from it, i dont see the issues
At least i dont understand the merge conflict purposes? Its just doesn't need to conflict with goons master branch

**Comment by cheekybrdy** [comment]
Please disregard that, review request was due to a accidental close and reopen

**Comment by cheekybrdy** [comment]
I've messed up this branch bigtime, I'll redo the PR

**Pull request closed without merging**